node_modules/npm-audit-report/reporters/detail.js

```diff
@@ -117,7 +117,7 @@ const report = function (data, options) {
               {'Package': advisory.module_name},
               {'Dependency of': `${resolution.path.split('>')[0]} ${resolution.dev ? '[dev]' : ''}`},
               {'Path': `${resolution.path.split('>').join(Utils.color(' > ', 'grey', config.withColor))}`},
-              {'More info': `https://nodesecurity.io/advisories/${advisory.id}`}
+              {'More info': advisory.url ? advisory.url : `https://nodesecurity.io/advisories/${advisory.id}`}
```

Comment:
This is identical, but shorter and avoids repetition:
```suggestion
              {'More info': advisory.url || `https://nodesecurity.io/advisories/${advisory.id}`}
```